src/util/requests.ts

```diff
@@ -0,0 +1,8 @@
+import { BidFilterStatus } from "../types";
+
+export function getBidFilterStatus(input : string | undefined): BidFilterStatus {
+    input = input ?? "";
```

Comment:
Consider creating a new variable here instead of mutating your argument param. It's good practice as introducing mutability to function arguments can cause unintended side effects elsewhere in your code.

Reply:
Sure, I can do that